src/js/components/stock-transfer/utils.jsx

```diff
@@ -94,3 +98,22 @@ export function prepareRequest(stockTransfer, status) {
 
   return { ...stockTransfer, stockTransferItems, status };
 }
+
+export const canEditRequest = (currentUser, request, location) => {
+  const isRequestApprovalSupported = supports(
+    location?.supportedActivities,
+    ActivityCode.APPROVE_REQUEST,
+  );
+  const isUserRequestor = currentUser?.id === request?.requestedBy?.id;
+
+  // If request approval is not supported by the location we are able to edit every request
+  if (!isRequestApprovalSupported) {
+    return true;
+  }
```

Comment:
nitpicky, but since we don't need `isUserRequestor` at this point yet, we should move either this `if` above the `isUserRequestor` or the `isUserRequestor` below this `if`.